lib/pages/community/foreign_community.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:lemmy_api_client/v3.dart';
+
+import '../../l10n/l10n.dart';
+import '../../stores/accounts_store.dart';
+import 'community.dart';
+
+class ForeignCommunityPage extends HookWidget {
```

Comment:
Could we name this like.. FederatedCommunityPage? I'd rather not introduce a new concept